SignalServiceKit/src/Messages/OWSReadReceiptManager.m

```diff
@@ -210,17 +210,7 @@ - (void)scheduleProcessing
 
             self.isProcessing = YES;
 
-            // Process read receipts every N seconds.
-            //
-            // We want a value high enough to allow us to effectively de-duplicate,
-            // read receipts without being so high that we risk not sending read
-            // receipts due to app exit.
-            const CGFloat kProcessingFrequencySeconds = 3.f;
-            dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(kProcessingFrequencySeconds * NSEC_PER_SEC)),
-                dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0),
-                ^{
-                    [self process];
-                });
+            [self process];
```

Comment:
We shouldn't delay "initial" read receipt processing, so that read receipts are emitted ASAP when user is interacting with a conversation.

We do want to wait between processing "passes" in cases where the user is reading a lot of messages in order to facilitate batching.